client/components/Layout.js

```diff
@@ -1,36 +1,46 @@
 import React from "react";
 import {Layout} from "antd";
-
 import Head from "./Head";
 import Sidebar from "./Sidebar";
 import Header from "./Header";
 import Footer from "./Footer";
+import Loading from "./Loading";
 
 const {Content} = Layout;
 
-export default props => (
-	<Layout>
-		<Head/>
-		<Sidebar/>
-		<Layout style={{
-			height: "100vh"
-		}}
-		>
-			<Header/>
-			<Content style={{
-				margin: "24px 16px 0"
-			}}
-			>
-				<div style={{
-					padding: 24,
-					background: "#fff",
-					minHeight: 360
-				}}
-				>
-					{props.children}
-				</div>
-			</Content>
-			<Footer/>
-		</Layout>
-	</Layout>
-);
+export default class extends React.Component {
```

Comment:
Shouldn’t this class be named (with something like `MainLayout`)?

Reply:
Good idea, I'll change the filename as well then.